tests/js/lib/request.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+var fluid = require("infusion");
+
+var kettle = require("kettle");
+kettle.loadTestingSupport();
+
+fluid.defaults("gpii.express.tests.request", {
```

Comment:
The latest commit addresses this, generic reusable bits are now in the `gpii.test` namespace, while specific testEnvironments and caseHolders are in the `gpii.tests` namespace.